CliClient/tests/test-utils.js

```diff
@@ -449,13 +504,16 @@ class TestApp extends BaseApplication {
 	}
 
 	async destroy() {
-		this.deinitRedux();
-		await this.waitForMiddleware_();
-		await super.destroy();
-
+		try {
+			await this.waitForMiddleware_();
+			this.deinitRedux();
+			await super.destroy();
+		} catch (e) {
```

Comment:
Any reason why the error is caught instead of letting the test fail with a stack trace?

Reply:
Fixed. (removed)